prometheus/templates/app_alert.rules.tmpl

```diff
@@ -0,0 +1,96 @@
+  - alert: High-Memory-Utilisation-${instance.app_name}
+    expr: avg by ( app ) (memory_utilization{app=~"${instance.app_name}"}) > ${instance.max_mem}
+    for: 5m
+    annotations:
+      summary:     ${instance.app_name} high memory utilization
+      dashboard:   ${cfapps_dashboard_url}&var-Applications=${instance.app_name}
+      description: "Memory utilization has increased in the last 5 minutes (current value: {{ $value }}%)"
+    labels:
+      severity:    high
+      app:         ${instance.app_name}
+      environment: ${instance.monitoring_space_name}
+%{ endfor ~}
+
+- name: Disk Utilisation
+  rules:
+%{ for instance in apps ~}
+  - alert: High-Disk-Utilisation-${instance.app_name}
+    expr: avg by ( app ) ( disk_utilization{ app=~"${instance.app_name}" }) > ${instance.max_disk}
+    for: 5m
+    annotations:
+      summary:     ${instance.app_name} high disk utilization
+      dashboard:   ${cfapps_dashboard_url}&var-Applications=${instance.app_name}
+      description: "Disk utilization has increased in the last 5 minutes (current value: {{ $value }})%"
+    labels:
+      severity:    high
+      app:         ${instance.app_name}
+      environment: ${instance.monitoring_space_name}
+%{ endfor ~}
+
+- name: App Crashes
+  rules:
+%{ for instance in apps ~}
+  - alert: App-Crash-${instance.app_name}
+    expr: rate(crash{app=~"${instance.app_name}"}[1m])*60 > ${instance.max_crash_count}
+    for: 5m
+    annotations:
+      summary:     At least one instance of ${instance.app_name} has crashed in the last 5 mins
+      dashboard:   ${cfapps_dashboard_url}&var-Applications=${instance.app_name}
+      description: At least one instance of ${instance.app_name} has crashed in the last 5 mins
+    labels:
+      severity:    high
+      app:         ${instance.app_name}
+      environment: ${instance.monitoring_space_name}
+%{ endfor ~}
+
+- name: Elevated Request Failures
+  rules:
+%{ for instance in apps ~}
+  - alert: Request-Failures-${instance.app_name}
+    expr: (sum(rate(requests{app="${instance.app_name}", status_range=~"5xx"}[5m]))) / (sum(rate(requests{app="${instance.app_name}"}[5m])) ) > ${instance.max_elevated_req_failure_count}
+    for: 5m
+    annotations:
+      summary:     High rate of failing requests
+      dashboard:   ${cfapps_dashboard_url}&var-Applications=${instance.app_name}
+      description: "The proportion of failed 5xx HTTP requests in the past 5 min is above 10% (current value: {{ humanizePercentage $value }})"
+    labels:
+      severity:    high
+      app:         ${instance.app_name}
+      environment: ${instance.monitoring_space_name}
+%{ endfor ~}
+
+- name: Average Response Time
+  rules:
+%{ for instance in apps ~}
+  - alert: Response-Times-${instance.app_name}
+    expr:  histogram_quantile(0.95, sum(rate(response_time_bucket{app="${instance.app_name}", status_range="2xx"}[5m])) by (le)) > %{ if instance.response_threshold == null }1%{ else }${instance.response_threshold}%{ endif }
```

Comment:
Could we set the default value in `try(settings.response_threshold, 1)` instead?